src/Frontend/Page/User.hs

```diff
@@ -356,7 +350,7 @@ createdIdeas userId ideasQuery = do
                     (mapM getIdeaStats
                      =<< filter visibleByCurrentUser
                          <$> findIdeasByUserId userId))
-        ds <- commonIdeaSpaceDelegations cUser user
+        ds <- commonIdeaSpaceDelegations cUser user >>= mapM delegationFull
```

Comment:
This could be resource consuming load. Loading the same user repeatedly. A function could collect all the user ids, load them, reuse the references.